versions/baseline.json

```
@@ -1325,8 +1325,8 @@
       "port-version": 0
     },
     "cnl": {
-      "baseline": "2019-06-23",
-      "port-version": 0
+      "baseline": "1.1.7",
+      "port-version": 1
```

```suggestion
      "port-version": 0
```
For this change, you can run `vcpkg x-add-version --overwrite-version cnl `to update.